corehq/apps/smsbillables/tasks.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from datetime import date
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.conf import settings
+
+from corehq.apps.smsbillables.models import (
+    SmsBillable,
+    SmsGatewayFeeCriteria,
+)
+from corehq.util.log import send_HTML_email
+from dimagi.utils.dates import add_months_to_date
+
+
+@periodic_task(run_every=crontab(day_of_month='1', hour=5, minute=0), queue='background_queue', acks_late=True)
+def send_gateway_fee_report_out():
+    backend_api_ids = SmsGatewayFeeCriteria.objects.values('backend_api_id').distinct()
+    first_day_previous_month = add_months_to_date(date.today(), -1)
+    billables_in_month = SmsBillable.objects.filter(
+        date_sent__year=first_day_previous_month.year,
+        date_sent__month=first_day_previous_month.month,
+    )
+    cost_by_backend = {
+        backend_api_id: sum(billable.gateway_charge for billable in
+            billables_in_month.filter(gateway_fee__criteria__backend_api_id=backend_api_id))
+        for backend_api_id in map(lambda _dict: _dict['backend_api_id'], backend_api_ids)
+    }
```

Comment:
You could probably write some sort of query aggregation to avoid querying in a loop and summing in memory, but I suppose there are few enough backend APIs that it doesn't really matter, yeah?

Reply:
What makes this complicated is that the gateway charge is the gateway fee times the currency conversion rate.  I imagine I would have to write some custom SQL to do this.  Right now, this runs fast enough, so I think it's okay.